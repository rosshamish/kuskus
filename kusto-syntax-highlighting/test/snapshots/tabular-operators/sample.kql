// https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/sampleoperator

let _data = range x from 1 to 100 step 1;
let _sample = _data | sample 1;
union (_sample), (_sample);

// https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/sampledistinctoperator

StormEvents | sample-distinct 10 of EpisodeId
