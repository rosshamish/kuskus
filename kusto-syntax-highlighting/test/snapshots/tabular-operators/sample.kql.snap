>// https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/sampleoperator
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.kusto comment.line.kusto
>
>let _data = range x from 1 to 100 step 1;
#^^^ source.kusto keyword.control.kusto
#   ^^^^^^^^^ source.kusto
#            ^^^^^ source.kusto meta.query.range.kusto keyword.other.query.kusto
#                 ^ source.kusto meta.query.range.kusto
#                  ^ source.kusto meta.query.range.kusto variable.other.column.kusto
#                   ^ source.kusto meta.query.range.kusto
#                    ^^^^ source.kusto meta.query.range.kusto keyword.other.operator.kusto
#                        ^ source.kusto meta.query.range.kusto
#                         ^ source.kusto meta.query.range.kusto constant.numeric.kusto
#                          ^ source.kusto meta.query.range.kusto
#                           ^^ source.kusto meta.query.range.kusto keyword.other.operator.kusto
#                             ^ source.kusto meta.query.range.kusto
#                              ^^^ source.kusto meta.query.range.kusto constant.numeric.kusto
#                                 ^ source.kusto meta.query.range.kusto
#                                  ^^^^ source.kusto meta.query.range.kusto keyword.other.operator.kusto
#                                      ^ source.kusto meta.query.range.kusto
#                                       ^ source.kusto meta.query.range.kusto constant.numeric.kusto
#                                        ^^ source.kusto
>let _sample = _data | sample 1;
#^^^ source.kusto keyword.control.kusto
#   ^^^^^^^^^^^^^^^^^^^ source.kusto
#                      ^^^^^^ source.kusto meta.query.sample.kusto keyword.other.query.kusto
#                            ^ source.kusto meta.query.sample.kusto
#                             ^ source.kusto meta.query.sample.kusto constant.numeric.kusto
#                              ^^ source.kusto
>union (_sample), (_sample);
#^^^^^ source.kusto keyword.other.query.kusto
#     ^^^^^^^^^^^^^^^^^^^^^^^ source.kusto
>
>// https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/sampledistinctoperator
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.kusto comment.line.kusto
>
>StormEvents | sample-distinct 10 of EpisodeId
#^^^^^^^^^^^^^^ source.kusto
#              ^^^^^^^^^^^^^^^ source.kusto meta.query.sample-distinct.kusto keyword.other.query.kusto
#                             ^ source.kusto meta.query.sample-distinct.kusto
#                              ^^ source.kusto meta.query.sample-distinct.kusto constant.numeric.kusto
#                                ^ source.kusto meta.query.sample-distinct.kusto
#                                 ^^ source.kusto meta.query.sample-distinct.kusto keyword.other.operator.kusto
#                                   ^ source.kusto meta.query.sample-distinct.kusto
#                                    ^^^^^^^^^ source.kusto meta.query.sample-distinct.kusto variable.other.column.kusto
>